docs/development/ide-configuration.md

```diff
@@ -38,6 +38,26 @@ When Gitpod IDE is started, the `npm install` command is launched automatically.
 
 Install the EditorConfig extension. A configuration file already exists in the repository, so it will apply right after the extension installation.
 
+#### [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)
+
+After the installation, you need to explicitly tell to the ESLint extension to watch typscript files for linting errors.
+  - Open the Command Palette (*`View -> Command Palette`* or *`Ctrl+Shift+P`* or *`Shift+Cmd+P`*)
+  - Type `Open Settings (JSON)` and select the option `Preferences: Open Settings (JSON)`
+  - Paste this code inside the opened JSON file
+
+  ```JSON
+    {
+      "eslint.validate": ["typescript", "typescriptreact"]
+    }
+  ```
+  - Open the Command Palette again
+  - Type `eslint`
+  - Select `ESLint: Enable ESLint`
```

Comment:
After the plugin installation, Eslint is enabled by default. So we don't need to do that 😉

Reply:
But, you can add this : 
<img width="578" alt="image" src="https://user-images.githubusercontent.com/4921914/95188660-0516c300-07cd-11eb-8069-a1c52b0fdae6.png">

Reply:
I don't have those commands, I suppose they come from an other plugin?

Reply:
It's the same. 
You can see the configuration of the extension here:
<img width="513" alt="image" src="https://user-images.githubusercontent.com/4921914/95310889-35259b00-088d-11eb-85ac-eb20727ee9ec.png">

<img width="1363" alt="image" src="https://user-images.githubusercontent.com/4921914/95310977-5090a600-088d-11eb-8d01-297a7d14c8ac.png">

Reply:
I see, it's not a command that's why i didn't find it.